Makefile

```diff
@@ -1,32 +1,40 @@
 
-bounce: build undeploy deploy
-
 build:
 	./gradlew build
 	docker build . -t hoptimator
 	docker build hoptimator-flink-runner -t hoptimator-flink-runner
 
+bounce: build undeploy deploy deploy-samples
+
 clean:
 	./gradlew clean
 
 undeploy:
 	kubectl delete -f ./deploy || echo "skipping"
+	kubectl delete configmap hoptimator-configmap || echo "skipping"
 
 quickstart: build deploy-dev-environment deploy
 
 deploy-dev-environment: 
 	kubectl create -f https://github.com/jetstack/cert-manager/releases/download/v1.8.2/cert-manager.yaml || echo "skipping"
-	helm repo add flink-operator-repo https://downloads.apache.org/flink/flink-kubernetes-operator-1.4.0/
-	helm upgrade --atomic --set webhook.create=false flink-kubernetes-operator flink-operator-repo/flink-kubernetes-operator || helm install --atomic --set webhook.create=false flink-kubernetes-operator flink-operator-repo/flink-kubernetes-operator
 	kubectl create namespace kafka || echo "skipping"
+	kubectl create namespace mysql || echo "skipping"
+	helm repo add flink-operator-repo https://downloads.apache.org/flink/flink-kubernetes-operator-1.4.0/
+	helm upgrade --install --atomic --set webhook.create=false flink-kubernetes-operator flink-operator-repo/flink-kubernetes-operator
 	kubectl apply -f "https://strimzi.io/install/latest?namespace=kafka" -n kafka
 	kubectl apply -f "https://strimzi.io/examples/latest/kafka/kafka-ephemeral-single.yaml" -n kafka
 	kubectl apply -f ./deploy/dev
 
-deploy:
+deploy: deploy-config
 	kubectl apply -f ./deploy/
 
+deploy-samples:
+	kubectl apply -f ./deploy/samples
+
+deploy-config:
+	kubectl create configmap hoptimator-configmap --from-file=model.yaml=test-model.yaml --dry-run=client -o yaml | kubectl apply -f -
```

Comment:
nit: Should the default file be called `test-model.yaml`? Is it setup like this to keep testing/local development simple?

It was removed from `Dockerfile` (this change), but it is still referenced as `test-model.yaml` in [HoptimatorOperatorApp](https://github.com/linkedin/Hoptimator/blob/041321a0c670ceb793e143263c45d4d461959d13/hoptimator-operator/src/main/java/com/linkedin/hoptimator/operator/HoptimatorOperatorApp.java#L31).